playbooks/oar_gcp_staging.yml

```diff
@@ -0,0 +1,24 @@
+---
+- hosts: gcp_oar_staging
+  remote_user: pulsys
+  become: true
+  vars:
+    - oar_post_install: |
+        Possible Things left to do:
+        - run a dsbuild, dsdeploy or dsbounce
```

Comment:
I am sorry, why is this not in place for the `gcp_oar_dev` playbook?

Reply:
when Atmire started working on this I stepped away. Will issue it and add in dataspace PR.